modules/tinymce/src/core/main/ts/api/EditorUpload.ts

```diff
@@ -239,11 +243,14 @@ const EditorUpload = (editor: Editor): EditorUpload => {
         return true;
       });
 
-      Arr.each(result, (resultItem) => {
-        replaceUrlInUndoStack(resultItem.image.src, resultItem.blobInfo.blobUri());
-        resultItem.image.src = resultItem.blobInfo.blobUri();
-        resultItem.image.removeAttribute('data-mce-src');
-      });
+      if (!Rtc.isRtc(editor)) {
```

Comment:
Is there any reason this one does a not condition and the others have the if/else? FWIW I'd prefer it this way

Reply:
because I did them at different times 🤷‍♂️ will fix